x/auction/client/cli/query.go

```diff
@@ -20,14 +20,15 @@ func GetQueryCmd(queryRoute string, cdc *codec.Codec) *cobra.Command {
 	}
 
 	auctionQueryCmd.AddCommand(client.GetCommands(
-		GetCmdGetAuctions(queryRoute, cdc),
+		QueryGetAuctionsCmd(queryRoute, cdc),
+		QueryParamsCmd(queryRoute, cdc),
 	)...)
 
 	return auctionQueryCmd
 }
 
-// GetCmdGetAuctions queries the auctions in the store
-func GetCmdGetAuctions(queryRoute string, cdc *codec.Codec) *cobra.Command {
+// QueryGetAuctionsCmd queries the auctions in the store
+func QueryGetAuctionsCmd(queryRoute string, cdc *codec.Codec) *cobra.Command {
 	return &cobra.Command{
 		Use:   "getauctions",
```

Comment:
should we rename this cmd? could just be `auctions`

Reply:
https://github.com/Kava-Labs/kava/pull/310/commits/b8c7ddb9717a8f3c67bfce9f15f936776705cb26